frontend/src/components/Tag/Tag.scss

```diff
@@ -1,12 +1,17 @@
 .Tag {
   display: inline-block;
-  margin: 5px;
-  font-size: 13px;
-  font-weight: 600;
-  padding: 10px 24px;
+  margin: 2px;
+  font-size: 15px;
+  font-weight: 500;
+  padding: 4px 17px;
```

Comment:
I'm not so sure about these changes. At minimum we should have a larger margin between the Tag elements. Check out the map page which uses the Tag component. The elements are too close together.